modules/loki-deployment-sizing.adoc

```diff
@@ -1,41 +1,38 @@
 // Module is included in the following assemblies:
-//cluster-logging-loki.adoc
+// * logging/cluster-logging-loki.adoc
+// * network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="loki-deployment-sizing_{context}"]
-= Deployment Sizing
+= Loki deployment sizing
 
-Sizing for Loki follows the format of `N<x>._<size>_` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
+Sizing for Loki follows the format of `<N>x.<size>` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
 
-[NOTE]
-====
-1x.extra-small is for demo purposes only, and is not supported.
-====
+ifeval::["{context}" == "cluster-logging-loki"]
+:restricted:
+endif::[]
 
 .Loki Sizing
 [options="header"]
 |========================================================================================
```

Comment:
This one is existing but also should technically be reformatted. I leave this to you, if you have bandwidth to overhaul it now or not.

Reply:
Great feedback, thank you! For the sake of completeness I've updated both tables.